tensorboardX/writer.py

```diff
@@ -450,7 +450,7 @@ def add_audio(self, tag, snd_tensor, global_step=None, sample_rate=44100):
             snd_tensor: :math:`(1, L)`. The values should lie between [-1, 1].
         """
         if self._check_caffe2(snd_tensor):
-            snd_tensor = workspace.FetchBlobl(snd_tensor)
```

Comment:
It that a typo? Please let someone familiar with caffe2 to check for such trivial error.

Reply:
Yes this was a typo